.cicd/create-docker-from-binary.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo ":docker::package: Push EOS ubuntu 18.04 base Image to Dockerhub"
+
+# buildkite-agent artifact download someversion.deb .
+
+echo ":download: Downloading artifact from Buildkite step Ubuntu 18.04"
+buildkite-agent artifact download '*.deb' --step ':ubuntu: Ubuntu 18.04 - Package Builder' .
+echo ":done: download successfull"
+
+PREFIX='base-ubuntu-18.04'
+SANITIZED_BRANCH=$(echo "$BUILDKITE_BRANCH" | sed 's.^/..' | sed 's/[:/]/_/g')
+SANITIZED_TAG=$(echo "$BUILDKITE_TAG" | sed 's.^/..' | tr '/' '_')
+echo "$SANITIZED_BRANCH"
+echo "$SANITIZED_TAG"
+
+# do docker build
+echo ":docker::build: Building image..."
+
+set -e
+IMAGE_ECR="$EOSIO_REGISTRY:$PREFIX-bin-$SANITIZED_BRANCH"
+DOCKERHUB_EOS_REGISTRY="docker.io/eosio/eos"
+IMAGE_DOCKER="$DOCKERHUB_EOS_REGISTRY:$PREFIX-bin-$SANITIZED_BRANCH"
+
+echo ":docker: Building Image...."
+docker build -t $IMAGE_ECR -t $IMAGE_DOCKER -f ./docker/dockerfile .
+echo "Build Done successfully!!"
+
+#do docker push to ECR
+echo "Pushing Image to ECR..."
+docker push $IMAGE_ECR
+echo ":done: Done"
+
+# do docker push Dockerhub
+echo "Pushing Image DockerHub EOSIO/EOS ..."
+docker push $IMAGE_DOCKER
+echo ":done: Done"
```

Comment:
Do we want this tag format of `:base-ubuntu-18.04-bin-$BRANCH` @cocho-gore , @masaka3620 , and @kj4ezj ? This seems kind of long to me.

Reply:
I think we can nix the `base-ubuntu-18.04-bin-` part.  I don't think we'll be creating different images per OS.  Let's stick with the branch/commit for now

Reply:
I already changed to only branch and tested.